src/main/java/com/rb/nonbiz/types/ClosedUnitFractionRange.java

```diff
@@ -144,8 +145,8 @@ public boolean isStrictlyLooser(ClosedUnitFractionRange other) {
    * </pre>
    */
   public boolean isSameOrLooser(ClosedUnitFractionRange other) {
-    int lowerComparison = rawRange.lowerEndpoint().compareTo(other.lowerEndpoint());
-    int upperComparison = rawRange.upperEndpoint().compareTo(other.upperEndpoint());
+    int lowerComparison = rawRange.lowerEndpoint().almostCompareTo(other.lowerEndpoint(), TINY_EPSILON_1e_12);
```

Comment:
this is the main change. A client had two values that were off by a super tiny epsilon (like 1e-15) due to double calculations.